src/it/unimi/dsi/fastutil/Function.java

```diff
@@ -40,22 +39,36 @@
  * @see java.util.Map
  */
 
-public interface Function<K,V> {
+public interface Function<K,V> extends java.util.function.Function<K,V> {
+
```

Comment:
Not sure if I entirely follow you here. You mean the `apply` default method? I think its documentation is fine as is. Maybe rewrite it to something like "returns the value associated with this key or null if no value is present, i.e. returns get(key)". The added default methods should have appropriate documentation.